index.html

```diff
@@ -4,9 +4,184 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Capstone</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/eca3ff88d7.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/home.css" />
+    <title>Melaku Eshetu Capstone project</title>
   </head>
   <body>
-    Capstone
+    <header class="navigation-bar">
+      <nav class="header-nav">
+        <ul class="list header-nav__list">
+          <li class="header-nav__item">
+            <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li class="header-nav__item">
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li class="header-nav__item"><a href="">English</a></li>
+          <li class="header-nav__item"><a href="">My page</a></li>
+          <li class="header-nav__item"><a href="">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="nav">
+        <a href="index.html">
+          <svg class="icon brand-icon">
+            <use xlink:href="./images/sprite.svg#remote-work"></use>
+          </svg>
+        </a>
+        <div class="nav__toggler">
+          <span class="toggler-bar first-bar"></span>
+          <span class="toggler-bar second-bar"></span>
+          <span class="toggler-bar third-bar"></span>
+        </div>
+        <ul class="list nav__list">
+          <li class="nav__item"><a href="index.html">Home</a></li>
+          <li class="nav__item"><a href="./about.html">About</a></li>
+          <li class="nav__item"><a href="#">Programs</a></li>
+          <li class="nav__item active-item"><a href="#">Contact Us</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="hero-block">
+      <span class="hero__title">"Hello Job seekers Land your Remote Job"</span>
+      <h1 class="hero__heading">
+        Level Up Your Career & Learn To Thrive In The Non-Coding Side Of Tech
+        with our free course.
+      </h1>
```

Comment:
- [ ] [REQUIRED] I would recommend that you remove the background image obstructing the `h1` tag for a better UI. 
### Your Design
![Screenshot from 2022-03-18 14-21-32](https://user-images.githubusercontent.com/58692590/159012036-d3f10ba9-3dcd-4f2e-9169-cc7a80c741a9.png)